GTScript-Parallel-model.md

```diff
@@ -103,3 +103,175 @@ for k in range(start, end):
         u[i, j, k] = 1
     parfor [i_start:i_end, j_start:j_end]:
         b[i, j, k] = u[i-2,j,k] + u[i+1,j,k] + u[i,j-1,k] + u[i,j-2,k]
+```
+
+## Conditionals
+
+GTScript supports 2 kinds of conditionals:
+
+- conditionals on scalar expressions or compile-time constants 
+- conditionals on field expressions (meaning a condition on the field value at the gridpoint of the current iteration or at a local offset of it)
+
+### Conditionals on scalar expressions or compile-time constants
+
+- Conditions in `if` statements are evaluated at run-time and are required to be of scalar type, e.g. scalar variables (not fields) or scalar expressions (no fields involved).
```

Comment:
gt4py also supports if-statements with compile-time evaluation.

Reply:
I would actually remove the whole "runtime" and "compile-time" narrative here, since it distracts from the actual specification of if-statements on scalars.